impl/server/pom.xml

```diff
@@ -97,23 +96,23 @@
         </exclusion>
       </exclusions>
     </dependency>
+
+    <!-- Needed for compilation -->
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-core</artifactId>
-      <version>${spring.version}</version>
-      <scope>provided</scope>
       <exclusions>
         <exclusion>
           <artifactId>*</artifactId>
           <groupId>*</groupId>
         </exclusion>
       </exclusions>
     </dependency>
+    <!-- -->
+
     <dependency>
       <groupId>org.springframework</groupId>
       <artifactId>spring-beans</artifactId>
-      <version>${spring.version}</version>
-      <scope>provided</scope>
```

Comment:
Doesn't removing the provided scope have an impact in the plugin assembly? Isn't spring included in the plugin lib?

Reply:
Not sure why a remove it, will test it, but is very likely to maintain provided here

Reply:
At first I kept provided scope because it was already there, and then I believe a did some testing on it and opted to keep provided. Since the outcome of this project will be a plugin deployed on the server, is most likely that spring-beans dependency is being provided by server context. Do you want me to invest some more testing on it?

Reply:
Spring jars aren't going to /lib folder of plugin